### DISCUSSION TITLE
_pywrap_file_io.RecursivelyCreateDir(compat.path_to_bytes(path)) tensorflow.python.framework.errors_impl.FailedPreconditionError: runs\train\exp is not a directory

### DISCUSSION THREAD

**Discussion by johnnyherbert3008** [ultralytics/yolov5#11663] [Q&A]
I'm trying to do a personalized training but I'm facing this error and I don't know how to solve it, can someone help me?

Python 3.9.13
Windows 11

**Comment by wwwhaoyuuu** [comment]
have you solved it？I am facing this problem now...

**Reply by wwwhaoyuuu** [reply]
ok , I have known. Because our project address contains illegal encoding.The solution is that you should move your project file into a right address..
